### PR TITLE
Ready for Go Live - FB + Twitter Meta Tags

### PR DESCRIPTION
**Pull request by loganbek**
I've added the initial version of the twitter + FB meta tags. I was able to use ngroc and my local copy to validate the twitter card but ran into issues with the facebook share debugger (from a quick search this seems like a common issue). 

Screen Shot 2019-12-14 at 9 50 50 PM

Screen Shot 2019-12-14 at 11 39 26 PM

There's still some work to be done, but I thought this would be a good point to give you an update and ask a few clarifying questions. 

1) You mention a page favicon. Are you looking for me to add one to the site? 
2) Is there any harm in using yarn deploy to test against the live app?

Thanks!

### CODE REVIEW

**Comment by owocki** [comment]
Thanks, looking like were on the right track!

> You mention a page favicon. Are you looking for me to add one to the site?

Yes pls

> Is there any harm in using yarn deploy to test against the live app?

donno but @abbey-titcomb might

**Comment by owocki** [comment]
@loganbek from the screenshot it looks like the twitter card image doesnt show up tho.. we'll want the logo to show up there (it might hae to be a png not a svg)

**Comment by loganbek** [comment]
@owocki I'll look into getting a favicon added and ensure the images work on both FB and twitter cards. @abbey-titcomb Let me know if I can update the live app for testing. These changes shouldn't cause any issues since I'm just adding meta tags to the header.

**Comment by owocki** [comment]
thanks @loganbek mind also throwing up a WIP PR for responsiveness fixes too?

**Comment by owocki** [comment]
can we get this out of WIP?  this is gonna go live tomo

**Comment by loganbek** [comment]
@owocki, this PR should be ready to go, although I'm still having trouble testing the FB card with the sharing debugger + ngrok. I've attached images of the favicon and the twitter card.

Screen Shot 2019-12-17 at 2 42 38 PM

Screen Shot 2019-12-17 at 2 45 20 PM

**Comment by abbey-titcomb** [comment]
Sorry @loganbek - just seeing this. Feel free to update live app for testing.

**Comment by owocki** [comment]
looks good to me!

File: components/Header.js [hunk 1]
```diff
@@ -12,12 +12,16 @@ export default function Header() {
...
+
+        <meta name="twitter:image" content="https://res.cloudinary.com/dvargvav9/image/upload/v1575999063/SW3_sngkyk.png" />
+
         <meta property="og:url" content="https://web3.sustainoss.org" />
         <meta property="og:type" content="website" />
         <meta property="og:title" content="Sustain Web3 : 2/13/20 in Boulder :: Checkout our LineUp" />
         <meta property="og:description" content="we are the architects, builders, designers, researchers building the Open Source web3. we believe that blockchain is a new hope for OSS Sustainability. we are working to build a world" />
-        <meta property="og:image" content="https://res.cloudinary.com/dvargvav9/image/upload/v1575999063/SW3_sngkyk.svg" />
+
+        <meta property="og:image" content="https://res.cloudinary.com/dvargvav9/image/upload/v1575999063/SW3_sngkyk.png" />
```
Comment (owocki) [inline components/Header.js:23]: hmm should we host this somewhere on web3.sustainoss.org? instead of the coders site?

**Pull request merged**